src/types/activity.ts

```diff
@@ -14,6 +14,7 @@ const pickSafely = <ObjectType>(keys: readonly `${string & keyof ObjectType}`[])
 
 export enum ResponderStatus {
   Unavailable = 0,
+  Cancel = 1,

```

Comment:
I think this commit has some big implications we need to think through. I don't understand what problem we're trying to solve.

The original intent was that `ResponderStatus` is a state, but this change seems to focus on `Cancel` as a state transition (from StandBy to Cancel). If we're going to track each state transition, I think this enum (and supporting code) will get quite messy.

If we say "SignedOut" is "no longer involved with the mission", what's wrong with moving from Standby ("I am interested / planning on going on this mission") to SignedOut?

If there's a concern that we need an explicit transition from StandBy to SignedOut, I'd be less worried about code that changes the text of the "Sign Out" action based on the participant's current state, but I feel like that's unnecessary complexity.

Reply:
I'm thinking of it more as "not responding"; as a state, not the state transition. That's actually why I didn't call it "Stand Down". I do think this is different than Signed Out, in terms of the timeline. "Cancel" was just what came to mind, but since you've prompted more thought, "NotResponding" could work, too.

I also feel like its a different state than "Unavailable", but, I'm not totally sure what the intent of unavailable was.

**This is how I'm thinking about each of the statuses:**
Not Responding (Cancel) means "I'm not currently responding to the mission"
Standby means "I am not currently responding to the mission, but I will be"
Signed In means "I'm currently responding to the mission"
Signed Out means "I was responding to the mission, but I'm not now"

The extra state of Not Responding provides a means to cancel a standby without adding an orphaned sign out to the history -- there will always be a corresponding sign in.

This has implications on the UI, and simplifies the necessary logic for things like the status indicator. A mission that you were on stand by but stood down for should not show signed out. Where as a mission that you signed into and out of, should.

Also, I think it makes a better user experience. Its more clear that I was never signed in; and I don't need the sign out dialog if I never responded.